### PR TITLE
HW_14

### PR DESCRIPTION
**Pull request by daevsikova**

### CODE REVIEW

**Comment by kraalfar** [comment]
1. при подсчете calc_distance нужно добавить размерности к векторам, чтобы посчиталась матрица. иначе он упадет
2. в calc_prototypes замени в return torch.Tensor на torch.vstack (или torch.cat с дополнительным dim)
3. в loss в недописанной часте, можно не писать цикл. можно query_labels как столбик приравнять к unique_labeles, получить матрицу, где в строчке по одному true и от нее аргмакс взять. а потом уже можно обычную кросс энтропию считать

можно доделать
